.grenrc.js

```diff
@@ -0,0 +1,44 @@
+module.exports = {
```

Comment:
How do we avoid duplicating this in every repo?

Reply:
Looking at the third party action we're using, it does next to nothing. Between the dockerfile and entrypoint,

https://github.com/lakto/gren-action/blob/master/entrypoint.sh
```
npm install github-release-notes@$GREN_VERSION -g
GREN_GITHUB_TOKEN=$GITHUB_TOKEN gren release --username=$username --repo=$repository $1 
```

Given that, I think it would make sense to just make our own action, which owns the expected args and config.

Reply:
Not sure about duplication as we are passing it to command we will need to have it somewhere accessible. If we can publicly host it somewhere, that might work. 

I was actually trying to build own action using commands and it was giving error. as this is not critical as of now and we might never end up changing the command considering most of the things are handled by config file, I ended up using this action.

Reply:
It's not really the command I'm concerned about, it's the config file. In our own action, we can mount our standard `.grenrc` file, run the command, then unmount it.